binding against incode dependency of lifecycle manager & various CLI alpha command improvements (#1544)
kyma-project/cli: docs/gen-docs/kyma_alpha_verify_module.md

---
title: kyma alpha verify module
---

Verifies the signature of a Kyma module bundled as an OCI container image.

## Synopsis

Use this command to verify a Kyma module.

### Detailed description

> Kyma modules can be cryptographically signed to make sure they are correct and distributed by a trusted authority. This command verifies the authenticity of a given module.

Reviewer: ```suggestion
Kyma modules can be cryptographically signed to ensure they are correct and distributed by a trusted authority. This command verifies the authenticity of a given module.
```

```bash
kyma alpha verify module --name MODULE_NAME --version MODULE_VERSION --registry MODULE_REGISTRY [flags]
```

## Flags

```bash

>   -c, --credentials string      Basic authentication credentials for the given registry in the format user:password

Reviewer: ```suggestion
  -c, --credentials string      Basic authentication credentials for the given registry in the user:password format.
```

      --insecure                Use an insecure connection to access the registry.

>       --key string              Specifies the path where the private key used for signing

Reviewer: ```suggestion
      --key string              Specifies the path where the private key is used for signing.
```

>       --name string             Name of the module

Reviewer: ```suggestion
      --name string             Name of the module.
```

>       --name-mapping string     Overrides the OCM Component Name Mapping, one of: "urlPath" or "sha256-digest" (default "urlPath")

Reviewer: ```suggestion
      --name-mapping string     Overrides the OCM Component Name Mapping. Use "urlPath" or "sha256-digest". (default "urlPath")
```
